specification/cognitiveservices/OpenAI.Inference/models/completions/azure_content_filtering.tsp

```diff
@@ -0,0 +1,175 @@
+  Content that displays explicit and severe harmful instructions, actions, 
+  damage, or abuse; includes endorsement, glorification, or promotion of severe 
+  harmful acts, extreme or illegal forms of harm, radicalization, or non-consensual 
+  power exchange or abuse.
+  """)
+  high: "high",
+}
+
+@added(ServiceApiVersions.v2023_06_01_Preview)
+@doc("Information about filtered content severity level and if it has been filtered or not.")
+model ContentFilterResult {
+  @doc("Ratings for the intensity and risk level of filtered content.")
+  @projectedName("json", "severity")
+  severity: ContentFilterSeverity;
+
+  @doc("A value indicating whether or not the content has been filtered.")
+  @projectedName("json", "filtered")
+  filtered: boolean;
+}
+
+alias ContentFilterResultsBase = {
+  @doc("""
+  Describes language related to anatomical organs and genitals, romantic relationships,
+   acts portrayed in erotic or affectionate terms, physical sexual acts, including 
+   those portrayed as an assault or a forced sexual violent act against one’s will, 
+   prostitution, pornography, and abuse.
+  """)
+  sexual?: ContentFilterResult;
+
+  @doc("""
+  Describes language related to physical actions intended to hurt, injure, damage, or 
+  kill someone or something; describes weapons, etc.
+  """)
+  violence?: ContentFilterResult;
+
+  @doc("""
+  Describes language attacks or uses that include pejorative or discriminatory language 
+  with reference to a person or identity group on the basis of certain differentiating 
+  attributes of these groups including but not limited to race, ethnicity, nationality,
+  gender identity and expression, sexual orientation, religion, immigration status, ability
+  status, personal appearance, and body size.
+  """)
+  hate?: ContentFilterResult;
+
+  @doc("""
+  Describes language related to physical actions intended to purposely hurt, injure,
+  or damage one’s body, or kill oneself.
+  """)
+  @projectedName("json", "self_harm")
+  selfHarm?: ContentFilterResult;
+
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  @doc("Describes whether profanity was detected.")
+  profanity?: ContentFilterDetectionResult;
+
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  @projectedName("json", "custom_blocklists")
+  @doc("Describes detection results against configured custom blocklists.")
+  customBlocklists?: ContentFilterBlocklistIdResult[];
+
+  @doc("""
+  Describes an error returned if the content filtering system is
+  down or otherwise unable to complete the operation in time.
+  """)
+  error?: Azure.Core.Foundations.Error;
+};
+
+@added(ServiceApiVersions.v2023_06_01_Preview)
+@doc("Information about content filtering evaluated against input data to Azure OpenAI.")
+model ContentFilterResultDetailsForPrompt {
+  ...ContentFilterResultsBase;
+
+  @doc("Whether a jailbreak attempt was detected in the prompt.")
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  jailbreak: ContentFilterDetectionResult;
```

Comment:
Is this always present for it to be required or is it a mistake?

Reply:
Great catch! I've always observed it in the actual responses thus far, but [per the spec](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2023-12-01-preview/inference.json#L1113) it's not listed as required. I'll update the TypeSpec here to reflect that.